compose.yml

```diff
@@ -121,5 +125,35 @@ services:
         max-file: '5'
         max-size: '10m'
 
+  ollama:
+    image: 'ollama/ollama:latest'
+    restart: always
+    healthcheck:
+      interval: 30s
+      retries: 5
+      test: ['CMD', 'ollama', 'list']
+      timeout: 10s
+      start_period: 60s
+    ports:
+      - '11434:11434'
+    volumes:
+      - '.data/ollama:/root/.ollama'
+    deploy:
+      restart_policy:
+        condition: on-failure
+      resources:
+        limits:
+          cpus: 2.0
```

Comment:
Why in compose.yml there are resource limitation available, but in compose.local.yml, not ?